### PR TITLE
Find Authors Ids Interface for semantic_scholar

### PR DESCRIPTION
**Pull request by Simnol22**

### CODE REVIEW

File: apps/find-authors-ids.py [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+        H.link(rel="stylesheet", href=here.parent / "paperoni" / "default.css")
+    )
+
+    author_name = "Amin Emad"
+
+    def confirm_id(auth, confirmed, auth_id):
+        link = auth.links[0].link
+        type = auth.links[0].type
+        if not confirmed:
+            link = "!" + auth.links[0].link
```
Comment (breuleux) [inline apps/find-authors-ids.py:136]: The ! is recognized on the type (e.g. `!semantic_scholar`) and not the link.

File: apps/find-authors-ids.py [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                query_name=ss.author_with_papers,
+            )
+            async for auth, result in results:
+                link = auth.links[0].link
+                if link not in tabIDS:
+                    tabIDS.append(link)
+                    author_name_area = H.div["authornamearea"](
+                        auth.name,
+                        H.br,
+                        H.br,
```
Comment (breuleux) [inline apps/find-authors-ids.py:231]: It would be better to handle spacing using styling (e.g. margin-top or margin-bottom on the appropriate elements).

**Comment by breuleux** [comment]
If you update starbear to the latest version, `page.query_params` should contain the query parameters (e.g. `?x=1&y=2` or `?author=john+smith`). I suggest to get the author from that.

File: apps/find-authors-ids.py (hunk 2 above)
Comment (Simnol22) [reply apps/find-authors-ids.py]: Fixed using `H.p` for information(name, link type), which automatically manages its spacing in [e5443de](https://github.com/mila-iqia/paperoni/pull/23/commits/e5443deae316e6b6bc180976d4c67fef3c80ac44)

**Comment by Simnol22** [comment]
> 
Adaptated with latest version & reorganization of webapps
[06789ed](https://github.com/mila-iqia/paperoni/pull/23/commits/06789ed5e1ac807a92ed0fc1791eb018dacb1a7f)

**Pull request merged**